src/rastervision/tagging/data/planet_kaggle.py

```diff
@@ -218,11 +226,28 @@ def __init__(self, datasets_path, options):
         download_dataset(PLANET_KAGGLE, self.file_names)
 
         self.dataset_path = join(datasets_path, PLANET_KAGGLE)
-        self.dev_path = join(self.dataset_path, self.dev_dir)
-        self.test_path = join(self.dataset_path, self.test_dir)
 
-        self.dev_file_inds = self.generate_file_inds(self.dev_path)
-        self.test_file_inds = self.generate_file_inds(self.test_path)
+        self.drop_file_inds = []
+        if self.drop_file_inds_file:
+            with open(join(self.dataset_path, self.drop_file_inds_file)) as f:
+                self.drop_file_inds = f.read().split('\n')
+
+        # dev_dir and test_ dir can contain multiple directories
+        # Assume file_inds are the same in every dev_dir and test_dir, respectively
+        if isinstance(self.dev_dir, list):
```

Comment:
There are several places where we test if something is a list and do one thing, and then another if it isn't. I think it would be simpler to make it so that `dev_dir` is always a list, and then we don't need two code paths. This means that wherever `dev_dir` was a singleton, we just replace with a list of length one.